scalafix-core/shared/src/main/scala/scalafix/internal/rule/Disable.scala

```diff
@@ -30,21 +30,34 @@ final case class Disable(index: SemanticdbIndex, config: DisableConfig)
       .getOrElse("disable", "Disable")(DisableConfig.default)
       .map(Disable(index, _))
 
-  override def check(ctx: RuleCtx): Seq[LintMessage] =
-    ctx.index.names.collect {
+  override def check(ctx: RuleCtx): Seq[LintMessage] = {
+    val buffer = List.newBuilder[LintMessage]
```

Comment:
Maybe make it immutable with `Option[LintMessage]` and `.flatMap`? Or performance is important here?

Reply:
Refactored to avoid builder.